consumer/src/main/kotlin/streams/kafka/KafkaAutoCommitEventConsumer.kt

```diff
@@ -91,18 +88,17 @@ open class KafkaAutoCommitEventConsumer(private val config: KafkaSinkConfigurati
                 try {
                     "ok" to it.toStreamsSinkEntity()
                 } catch (e: Exception) {
-                    "error" to DLQData.from(it, e, this::class.java)
+                    "error" to ErrorData.from(it, e, this::class.java)
                 }
             }
             .groupBy({ it.first }, { it.second })
             .let {
-                it.getOrDefault("error", emptyList<DLQData>())
-                        .forEach{ sentToDLQ(it as DLQData) }
+                it.get("error")?.let { handleErrors(it as List<ErrorData>) }
                 it.getOrDefault("ok", emptyList()) as List<StreamsSinkEntity>
             }
 
-    private fun sentToDLQ(dlqData: DLQData) {
-        dlqService?.send(config.streamsSinkConfiguration.dlqTopic, dlqData)
+    private fun handleErrors(errorData: List<ErrorData>) {
```

Comment:
If all this function does is call a different method on errorService I don't see the point of it.  Just remove this function and inline the use of errorService.report

Reply:
ya, it used to have more code.